src/styles/main.scss

```
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import "./reset.scss";
+
+@import "./normalize.scss";
+
+@import "./utils/variables";
+
+@import "./blocks/days.scss";
```

```suggestion
@import "./reset.scss";
@import "./normalize.scss";
@import "./utils/variables";
@import "./blocks/days.scss";
```

You can omit `.scss` in imports